CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ NOTE: Support for syncing with realm.cloud.io and/or Realm Object Server has bee
 NOTE: This version bumps the Realm file format to version 11. It is not possible to downgrade to earlier versions. Older files will automatically be upgraded to the new file format. Files created by Realm JavaScript prior to v1.0.0, might not be upgradeable. Only [Realm Studio 10.0.0](https://github.com/realm/realm-studio/releases/tag/v10.0.0-beta.1) or later will be able to open the new file format.
 
 ### Enhancements
-* None.
+* Adding the possibility of passing a string argument when constructing an App. ([#3082](https://github.com/realm/realm-js/pull/3082))
```

Comment:
I thought GFM auto-linkified things like `#3082` (eg #3082). I think it does something fancier than a plain link, so it may be better to let it do its thing than manually linkify it here.

Reply:
Also, it may be better to link to the issue resolved (#3081) rather than the PR resolving it.

Reply:
I'm primarily just following past usage here.
Also, historically we've pasted the top part of the changelog when publishing a release on Slack, where these links translate better than if they're just plain `#3082` text.